src/InteractionModNborhood.H

```diff
@@ -30,28 +31,36 @@ static void binaryInteractionNborhood ( const int a_i, /*!< Index of infectious
     auto nborhood_ptr = a_ptd.m_idata[IntIdx::nborhood];
     auto school_ptr = a_ptd.m_idata[IntIdx::school];
     auto withdrawn_ptr = a_ptd.m_idata[IntIdx::withdrawn];
+    auto work_nborhood_ptr = a_ptd.m_idata[IntIdx::work_nborhood];
 
     if (withdrawn_ptr[a_i] || withdrawn_ptr[a_j]) { return; }
 
     //infect *= i_mask;
     //infect *= j_mask;
     ParticleReal prob = 1.0_prt;
 
-    // school < 0 means a child normally attends school, but not today
-    /* Should always be in the same community = same cell */
-    if (school_ptr[a_i] < 0) {  // not attending school, use _SC contacts
-        prob *= 1.0_prt - infect * a_lparm->xmit_comm_SC[age_group_ptr[a_j]] * a_social_scale;
-    } else {
-        prob *= 1.0_prt - infect * a_lparm->xmit_comm[age_group_ptr[a_j]] * a_social_scale;
+    int nborhood_ai = nborhood_ptr[a_i];
+    int nborhood_aj = nborhood_ptr[a_j];
+
+    if (m_at_work_ptr) {
```

Comment:
Change this to `a_at_work_ptr`. `a_` is used to denote an incoming argument, `m_` is used to denote a member variable in a class/struct.